src/azure/Templates/AzurePagingMethodTemplate.cshtml

```diff
@@ -113,7 +113,13 @@ else
         def internal_paging(next_link=None):
             request = prepare_request(next_link)
 @EmptyLine
-            response = self._client.send(request, stream=False, **operation_config)
+            try:
+                pipeline_response = self.pipeline.run(request)
+                response = pipeline_response.http_response.internal_response
+        
+            finally:
+                if not self._config.connection.keep_alive and (not response):
+                    self.pipeline._sender.driver.session.close()
```

Comment:
We don't need this `finally` clause as it has been added to the transport layer in azure-core:
https://github.com/Azure/azure-sdk-for-python/blob/azure-core/azure-core/azure/core/pipeline/transport/requests_basic.py#L212

Additionally, a pipeline not longer has an `_sender,driver` so this will raise an error.